### PR TITLE
fix(webapp): add new user column and update its value

### LINKED ISSUE #497: The token needs to be reactive
Related to: https://github.com/eoscostarica/lifebank/projects/1#card-51684384

Steps to test (Under lifebank and donor profile):

**Sequence**: When a Lifebank transfer tokens to a donor, this should reflect automatically inside the Donor Profile Dashboard.

1. Go to dashboard section
2. See reflected current tokens

Solutions:
1. Use hasura subscription
2. Use blockchain approach with dmobs (make subcriptions). References https://github.com/EOSIO/history-tools

- [x] Database scheme design migrations
- [x] Hasura action for token transfers
- [x] Persist token balance database
- [x] Graphql subscriptions to reflect balance on front-end

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #497 

### What does this PR do?
Add new user column and updates its value coming from a Lifebank transfer.

### How should this be tested?
- Make run
- Go to Lifebank token transfer
- Trans tokens to a specific value

### CODE REVIEW

File: hapi/src/api/account.api.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ const createLifebank = async ({
   }
 }
 
-const getProfile = async account => {
+const getProfile = async (account) => {
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:138]: This is coming form the CodeFactor, not sure about the reason, but it is what's happening

**Review by JustinCast — APPROVED** [review]

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by steph222 — APPROVED** [review]

**Review by AngeloCG97 — APPROVED** [review]

File: webapp/src/components/DonationsDashboard/DonationsDashboard.js [hunk 2]
```diff
@@ -398,8 +403,12 @@ const DonationsDashboard = ({ isDesktop, currentUser, isOffer }) => {
   }, [currentUser, history, client, loadProfile, state, open, transferResult])
 
   useEffect(() => {
-    setTokens(role === "donor" && profile?.balance.length ? profile.balance.join(',').split(' ')[0] : 0)
-  }, [profile])
+    if(tokenUser.user) {
```
Comment (JustinCast) [inline webapp/src/components/DonationsDashboard/DonationsDashboard.js:406]: When it just one single line in the body of a comparison, you can omit the curly braces

File: webapp/src/components/DonationsDashboard/DonationsDashboard.js [hunk 3]
```diff
@@ -398,8 +403,12 @@ const DonationsDashboard = ({ isDesktop, currentUser, isOffer }) => {
...
+      setTokens(parseInt(tokenUser.user[0].token))
+    } else {
+      setTokens(role === "donor" && profile?.balance.length ? profile.balance.join(',').split(' ')[0] : 0)
```
Comment (JustinCast) [inline webapp/src/components/DonationsDashboard/DonationsDashboard.js:409]: Same here

File: webapp/src/components/DonationsDashboard/DonationsDashboard.js (hunk 2 above)
Comment (leisterfrancisco) [reply webapp/src/components/DonationsDashboard/DonationsDashboard.js:406]: @JustinCast I'm just following this standard, is it right or I should to change it?

image

Comment (JustinCast) [reply webapp/src/components/DonationsDashboard/DonationsDashboard.js:406]: @leisterfrancisco you can use the first one

**Pull request merged**